gtk/src/dark/gtk-3.0/meson.build

```diff
@@ -5,18 +5,18 @@ gtk3_asset_dir = join_paths(gtk3_dir, 'assets')
 
 # Dependencies of the files that need to be compiled
 gtk3_scss_dependencies = [
-  '../../light/gtk-3.0/_colors-public.scss',
-  '../../light/gtk-3.0/_ubuntu-colors.scss',
-  '../../light/gtk-3.0/_colors.scss',
-  '../../light/gtk-3.0/_common.scss',
-  '../../light/gtk-3.0/_apps.scss',
-  '../../light/gtk-3.0/_drawing.scss',
+  '../../light/gtk-3.20/_colors-public.scss',
```

Comment:
I've updated the dark theme build to produce `gtk-3.0` and `gtk-3.20` dirs.  Also, rather than rebuilding the CSS, I changed it to just symlink to the light theme's `gtk-dark.css`.